src/citrine/resources/table.py

```diff
@@ -111,6 +111,35 @@ def build_versions(collection: Iterable[dict]) -> Iterable[Table]:
 
         return self._paginator.paginate(fetch_versions, build_versions, page, per_page)
 
+    def list_for_ara_definition(self,
```

Comment:
Can we change to `list_by_ara_definition`?

Reply:
I think we want this to be `list_by_config` if we're moving forward with the new terminology

Reply:
I agree with this

Reply:
Good call. Fixed.